upgrade/upgrade-scripts/execute

```diff
@@ -192,8 +192,13 @@ systemctl mask docker.service
 # release that suffers from this defect; we've dis-associated the "etab"
 # file from the "nfs-kernel-server" package in all later releases.
 #
+# As of release 7.0.0.0, we also need to preserve the "rmtab" file so all
```

Comment:
This looks good, but the comment right above says that the code block can be removed once we no longer support upgrading from 6.0. This is extremely confusing because we already do not support upgrading from 6.0.0.0. Two related thoughts:
1. Someone might come through and delete your new code thinking that it's no longer needed
2. The comment needs to be updated
3. The etab code can be removed since we don't support upgrading from 6.0.0.0

Reply:
3. Yes, I think you're right..

2. The comment comment might be more clear after etab references are removed, too

Reply:
Good point!  I updated the comments to be `rmtab` based and remove the `etab` code line.